azext_iot/_help.py

```diff
@@ -1238,3 +1238,18 @@
     type: command
     short-summary: Delete a device registration in an Azure IoT Hub Device Provisioning Service.
 """
+
+helps[
+    "iot dps device-key"
+] = """
+    type: group
+    short-summary: Create device SAS key
+"""
+
+helps[
+    "iot dps device-key create"
+] = """
+    type: command
+    short-summary: Generate a device SAS key.
+    Long-summary: Generate device key from primary group SAS token.
```

Comment:
The language should be around generating a derived device key from a [DPS enrollment group SAS token](https://docs.microsoft.com/en-us/azure/iot-dps/concepts-symmetric-key-attestation#group-enrollments)